content/blog/oddnews/oddnews-21.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CSS Cascade Layers
+issue: 21
+url: http://eepurl.com/i_NTYw
```

Comment:
I think `url` is only used if we also have a `venue` which would make sense for these posts.